src/components/Projects/MyProjects.tsx

```diff
@@ -0,0 +1,68 @@
+import { t, Trans } from '@lingui/macro'
+import Grid from 'components/shared/Grid'
+import Loading from 'components/shared/Loading'
+import ProjectCard from 'components/shared/ProjectCard'
+import { NetworkContext } from 'contexts/networkContext'
+import { ThemeContext } from 'contexts/themeContext'
+import { useMyProjectsQuery } from 'hooks/v1/Projects'
+import React, { useContext } from 'react'
+import { InfoCircleOutlined } from '@ant-design/icons'
+
+export default function MyProjects() {
+  const { userAddress } = useContext(NetworkContext)
+
+  const { data: projects, isLoading } = useMyProjectsQuery(userAddress)
+
+  const {
+    theme: { colors },
+  } = useContext(ThemeContext)
+
+  return (
+    <React.Fragment>
```

Comment:
**suggestion**: wondering if we should also add a small note above the projects, something like "Juicebox projects you own". Not sure though

Reply:
I modeled after the 'Holdings' tab, there is an info paragraph below that says 'Projects that you have created'.

We could put it above for clarity's sake in both tabs?

Reply:
@jmilldotdev nice nice, yep thats great, i think keep it as you have it.

Just left one comment about hiding it on the empty state: https://github.com/jbx-protocol/juice-interface/pull/937/files#r867260015